changelog.txt

```diff
@@ -980,8 +980,8 @@ This version does include important changes in database. More than ever, make a
 = 0.9.4 (2012-10-23) =
 
 * Add Afrikaans translation contributed by [Kobus Joubert](http://translate3d.com/)
-* Add Belarusian translation contributed by [Alexander Markevitch](http://fourfeathers.by/)
-* Add Afrikaans (af) and Belarusian (be_BY) to predefined languages list (automatic download and update of language files won't work)
+* Add Belorussian translation contributed by [Alexander Markevitch](http://fourfeathers.by/)
+* Add Afrikaans (af) and Belorussian (be_BY) to predefined languages list (automatic download and update of language files won't work)
```

Comment:
I believe that Belarusian is the correct word: https://en.wikipedia.org/wiki/Belarusian_language